tutor/src/models/student-tasks.js

```diff
@@ -48,9 +49,10 @@ export class CourseStudentTasks extends Map {
   }
 
   // note: the response also contains limited course and role information but they're currently unused
-  onLoaded({ data: { tasks, research_surveys } }) {
+  onLoaded({ data: { tasks, research_surveys, all_tasks_are_ready } }) {
     this.researchSurveys = research_surveys ? new ResearchSurveys(research_surveys) : null;
     this.mergeModelData(tasks);
+    this.all_tasks_are_ready = all_tasks_are_ready;
   }
```

Comment:
This is the bug, we never actually updated the flag when the network request finishes.  On normal models we don't need to do that but this is a Map so it doesn't do that for us.